lib/middleman-hashicorp/partials/_enterprise-alert.html.erb

```diff
@@ -0,0 +1,12 @@
+<notextile>
```

Comment:
The thing I'm worried about is how older, non-HTML5 browsers will respond to this. Will they just not render the content inside? What is the browser support for this tag?